src/components/shared/col/col.styles.js

```diff
@@ -0,0 +1,19 @@
+import { createUseStyles } from 'react-jss'
+
+const useColStyles = createUseStyles(theme => ({
+  col: {
```

Comment:
missing colWrapped styles used in the jsx?

Reply:
Actually neither Col nor Row components are used. I removed them now, so this is not an issue any more.